lib/fastlane/actions/swiftlint.rb

```diff
@@ -5,7 +5,21 @@ def self.run(params)
         if `which swiftlint`.to_s.length == 0 and !Helper.test?
           raise "You have to install swiftlint using `brew install swiftlint`".red
         end
-        Actions.sh("swiftlint")
+
+        command_prefix = [
+          'cd',
+          File.expand_path('.').shellescape,
+          '&&'
+        ].join(' ')
+        swiftlint_args = "> #{params[:report_file]}"
```

Comment:
The check that the variable is set is implemented via FastlaneCore::ConfigItem.
In lines 38-40 the variable is configured as non-optional.
So this is safe.
But one could argue whether this variable should be optional. I dunno.